src/blocks/cards.scss

```diff
@@ -0,0 +1,101 @@
+.card {
+  // height: 406px;
+
+  width: 200px;
+  background-color: $four-color;
```

Comment:
Ensure that the variable `$four-color` is defined in your project. If it's not defined, this will cause a compilation error.